package.json

```diff
@@ -36,8 +36,8 @@
     "browserify": "node_modules/browserify/bin/cmd.js test/*.js --outfile lib/test/bundle.js",
     "bookmarklet": "scripts/bookmarklet",
     "clean": "rimraf dist",
-    "jscs": "jscs scripts/bookmarklet scripts/build **/*.js",
-    "jshint": "jshint scripts/bookmarklet scripts/build **/*.js",
+    "jscs": "jscs scripts/bookmarklet scripts/build **/*.js **/**/*.js",
```

Comment:
I don't think so.  I mean to test it last night at home.  But I think that means for both jscs and jshint: "Search starting here for any files ending in '.js`."

I did test yesterday on a Windows 7 machine and it worked fine.